templates/portals.example.yml

```
@@ -1197,3 +1197,63 @@ tracked_companies:
+  - name: SolidJobs Finances
+    careers_url: https://solid.jobs/public-api/offers/finances?campaign=career-ops
+    provider: solidjobs
+    enabled: false
+    notes: "Polish job board — Finances division"
+
+  - name: SolidJobs Other
+    careers_url: https://solid.jobs/public-api/offers/other?campaign=career-ops
+    provider: solidjobs
+    enabled: false
+    notes: "Polish job board — Other specialists division"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)